STYLING_EXAMPLES.md

```
@@ -0,0 +1,617 @@
+---
+
+## 🔘 Buttons <a name="buttons"></a>
+
+### Basic Buttons
+
+```python
+from shiny import ui
+
+# Primary button (blue, filled)
+ui.input_action_button("btn_submit", "📋 Submit", class_="btn-primary")
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)